### PR TITLE
updated heroku db

### PR DESCRIPTION
**Pull request by TanyaSilyutina**

### CODE REVIEW

File: db/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (fevercatdream) [inline db/index.js:1]: What is this code for?

File: controllers/accounts.js [hunk 2]
```diff
@@ -66,6 +66,7 @@ router.post('/token', async (req, res) => {
     }
 
     console.log(req.body);
+    console.log(req.dne.asdf.s.df.sdf.dsf);
```
Comment (fevercatdream) [inline controllers/accounts.js:69]: What is this?

File: package.json [hunk 3]
```diff
@@ -7,6 +7,9 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "watch": "nodemon index.js",
     "seed": "node seeds/seed.js",
+    "seed_heroku": "dotenv -e .env.jaws -- node seeds/seed.js",
```
Comment (fevercatdream) [inline package.json:10]: What goes into .env.jaws?

File: package.json [hunk 4]
```diff
@@ -7,6 +7,9 @@
...
+    "syncdb_local": "node db/index.js",
```
Comment (fevercatdream) [inline package.json:11]: How does this work?

**Review by fevercatdream — CHANGES_REQUESTED** [review]
I have added some questions regarding some code here.

File: db/index.js (hunk 1 above)
Comment (TanyaSilyutina) [reply db/index.js:1]: it restarts the db when I want to restart it. I'd want to restart it if i make changes to models

File: controllers/accounts.js (hunk 2 above)
Comment (TanyaSilyutina) [reply controllers/accounts.js:69]: this was a test line, i forgot to delete it

File: package.json (hunk 3 above)
Comment (TanyaSilyutina) [reply package.json:10]: our db variables. you don't need it for the app to work. i need them to restart the db when needed

File: package.json (hunk 4 above)
Comment (TanyaSilyutina) [reply package.json:11]: force: true is isolated to a separate file to restart the db in a controlled way

**Pull request merged**